site/_scss/blocks/_sample-item.scss

```diff
@@ -0,0 +1,35 @@
+.no-js {
+  .sample-item__collapse-button {
+    display: none;
+  }
+
+  .show-more-button {
+    display: none;
+  }
+}
+
+.sample-item {
+  margin-bottom: get-size(200);
```

Comment:
Thoughts on increasing these two (600 maybe?). I think that works better with the current text size.

Reply:
600 seems a bit too large, maybe 400 on large screens and 200 on mobile?
![image](https://github.com/GoogleChrome/developer.chrome.com/assets/18638914/6e69eb1b-00c3-494b-abbc-08b56bed4b78)
![image](https://github.com/GoogleChrome/developer.chrome.com/assets/18638914/66d51f44-8806-4787-836d-424e8f6f16a5)

Reply:
That looks better 👍